webviz_subsurface/_datainput/eclipse_init_io/pvt_common.py

```diff
@@ -74,66 +75,85 @@ def __init__(self) -> None:
         """Base implementation, raises a NotImplementedError."""
         raise NotImplementedError("You must not create objects of this base class.")
```

Comment:
We could perhaps now remove the `__init__` alltogether in the ABC? See example snippet below.